src/renderers/WebGLRenderer.js

```diff
@@ -2392,6 +2395,13 @@ class WebGLRenderer {
 
 		};
 
+		this.resetColorManagement = function () {
+
+			_gl.drawingBufferColorSpace = this._outputColorSpace === DisplayP3ColorSpace ? 'display-p3' : 'srgb';
+			_gl.unpackColorSpace = ColorManagement.workingColorSpace === LinearDisplayP3ColorSpace ? 'display-p3' : 'srgb';
+
+		};
+
```

Comment:
Do we need this new method?
Could we inline the calls in `set outputColorSpace( colorSpace )`?

Reply:
No need for it – or we don't need a public method, at least. I've inlined the calls in the setter. ✅